src/hip/hip_build_utils.cpp

```diff
@@ -137,6 +137,26 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
             MIOPEN_LOG_E("failed to find *.hsaco in " << hsaco->path().string());
         }
 
+        return hsaco->path();
+    }
+#elif OFFLOADBUNDLER_BIN
+    if(IsClangXXCompiler())
+    {
+        // call clang-offload-bundler
+        tmp_dir->Execute(OFFLOADBUNDLER_BIN,
+                         "-type=o -targets=hip-amdgcn-amd-amdhsa-" + dev_name + " -inputs=" +
```

Comment:
all options need `--` instead of `-`.

also, you must have an `--outputs=` option.

Reply:
Ok, thanks.
The entirety of my offload-bundler knowledge comes from the one line Aaron provided in the JIRA.

Reply:
sorry. actually `-` also works. but you still need `-outputs=` option.

Reply:
done